alert-common/src/main/java/com/synopsys/integration/alert/common/action/api/GlobalConfigurationModelToConcreteConverter.java

```diff
@@ -9,9 +9,36 @@
 
 import java.util.Optional;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.synopsys.integration.alert.api.common.model.ValidationResponseModel;
+import com.synopsys.integration.alert.api.common.model.errors.AlertFieldStatus;
 import com.synopsys.integration.alert.common.persistence.model.ConfigurationModel;
 import com.synopsys.integration.alert.common.rest.model.ConfigWithMetadata;
 
-public interface GlobalConfigurationModelToConcreteConverter<T extends ConfigWithMetadata> {
-    Optional<T> convert(ConfigurationModel globalConfigurationModel);
+public abstract class GlobalConfigurationModelToConcreteConverter<T extends ConfigWithMetadata> {
```

Comment:
The majority of the relevant changes are in this class. Everything else is primarily implementation